<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MembreAbonnementWindow</class>
 <widget class="QMainWindow" name="MembreAbonnementWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="amTableWidget">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>60</y>
      <width>311</width>
      <height>401</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Abonnement</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Debut</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Fin</string>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="widgetDetail" native="true">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>291</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gabriola</family>
       <pointsize>26</pointsize>
       <bold>false</bold>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>Détail du membre</string>
     </property>
    </widget>
    <widget class="QLabel" name="nom_dl">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gabriola</family>
       <pointsize>18</pointsize>
       <bold>false</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>RAHARISON</string>
     </property>
    </widget>
    <widget class="QLabel" name="prenom_dl">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gabriola</family>
       <pointsize>18</pointsize>
       <bold>false</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>Hajaniaina Marjean</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gabriola</family>
       <pointsize>18</pointsize>
       <bold>false</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>Prénom :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gabriola</family>
       <pointsize>18</pointsize>
       <bold>false</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>Nom :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gabriola</family>
       <pointsize>18</pointsize>
       <bold>false</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>Née le :</string>
     </property>
    </widget>
    <widget class="QLabel" name="naissance_dl">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>100</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Handwriting</family>
       <pointsize>11</pointsize>
       <bold>false</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>05 Juillet 1998</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gabriola</family>
       <pointsize>18</pointsize>
       <bold>false</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>Contact :</string>
     </property>
    </widget>
    <widget class="QLabel" name="contact_dl">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>130</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Handwriting</family>
       <pointsize>11</pointsize>
       <bold>false</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>+261 34 07 211 10</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gabriola</family>
       <pointsize>18</pointsize>
       <bold>false</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>Adresse :</string>
     </property>
    </widget>
    <widget class="QLabel" name="adresse_dl">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Handwriting</family>
       <pointsize>9</pointsize>
       <bold>false</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>Lot IVX 75E bis Ankazomanga Sud</string>
     </property>
    </widget>
    <widget class="QPushButton" name="editButton">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>220</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Modifier</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widgetEdit" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>291</width>
      <height>351</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gabriola</family>
       <pointsize>26</pointsize>
       <bold>false</bold>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>Modification</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gabriola</family>
       <pointsize>18</pointsize>
       <bold>false</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>Contact :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_26">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gabriola</family>
       <pointsize>18</pointsize>
       <bold>false</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>Adresse :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gabriola</family>
       <pointsize>18</pointsize>
       <bold>false</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>Prénom :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_27">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gabriola</family>
       <pointsize>18</pointsize>
       <bold>false</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>Nom :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_28">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gabriola</family>
       <pointsize>18</pointsize>
       <bold>false</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>Née le :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nomEdit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>RAKOTOMANDRINDRA</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="prenomEdit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>100</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Mahery Tsiory Raphaël</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateNaissanceEdit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>150</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="contactEdit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>200</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>+261 34 07 211 10</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="adresseEdit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>250</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Lot IVX 75 E bis Ankazomanga Sud</string>
     </property>
    </widget>
    <widget class="QPushButton" name="confirmEditButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Enregistrer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cancelEditButton">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>300</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Annuler</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="newAbonnementWidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>291</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QComboBox" name="abonnementComboBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>11</pointsize>
       <italic>true</italic>
       <bold>true</bold>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>Enregistrer un abonnement</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gabriola</family>
       <pointsize>18</pointsize>
       <bold>false</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>Début :</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEdit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>100</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="newAbonnementButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gabriola</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Enregistrer l'abonnement</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gabriola</family>
      <pointsize>26</pointsize>
      <bold>false</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Liste des abonnements</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-1</y>
      <width>641</width>
      <height>491</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/res/img/background.png);</string>
    </property>
   </widget>
   <zorder>widget</zorder>
   <zorder>amTableWidget</zorder>
   <zorder>label_5</zorder>
   <zorder>widgetEdit</zorder>
   <zorder>newAbonnementWidget</zorder>
   <zorder>widgetDetail</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
